content/concept-checks.md

```diff
@@ -0,0 +1,97 @@
+
+Unlike traditional checks, XRPL checks have expiration dates after which they may no longer be cashed. This is to prevent an unusable Check object from cluttering the Ledger. If the receiver doesn't successfully cash the Check before it expires, the Check object remains in the XRP Ledger until someone cancels it. Anyone may cancel the Check after it expires. Only the sender and receiver can cancel the Check before it expires or is cashed. The Check object is removed from the Ledger when the sender cashes the check or someone cancels it.
+
+Checks are similar to [Escrow](https://ripple.com/build/escrow/#escrow) and [Payment Channels](https://ripple.com/build/payment-channels-tutorial/), two other asynchronous transfer methods available on the XRP Ledger. There are two important differences between those methods and Checks:
+
+* You can send issued currency with Checks. You can only use XRP with Payment Channels and Escrow.
+
+* Checks do not tie up any funds. The XRP involved in Payment Channels and Escrow cannot be spent after until it is released by the sender (Payment Channels), an expiration, or crypto-condition (Escrow).
+
+
+***Note:*** The [Checks](https://ripple.com/build/known-amendments/#checks) amendment changes the [OfferCreate transaction](https://ripple.com/build/transactions/#offercreate) to return `tecEXPIRED` when trying to create an Offer whose expiration time is in the past. Without this amendment, an OfferCreate whose expiration time is in the past returns `tesSUCCESS` but does not create or execute an Offer.
+
+## Why Checks?
+
+Traditional checks allow people to transfer balances without immediately exchanging physical currency. XRPL Checks allow people to exchange funds asynchronously using a process that is familiar to and accepted by the banking industry.
+
+XRPL Checks also solve a problem that is unique to the XRP Ledger: they allow users to reject unwanted payments or accept only a portion of a payment. This is useful for institutions that need to be careful about accepting payments for compliance reasons.
+
+Checks potentially enable many other use cases. Ripple encourages the community to find new and creative applications for Checks.
+
+
+### Use Case: Payment Authorization
+
+**Problem:** To comply with regulations like [BSA, KYC, AML, and CFT](https://ripple.com/build/gateway-guide/#gateway-compliance), financial institutions must provide documentation about the source of funds they receive. Such regulations seek to prevent the illicit transfer of funds by requiring institutions to disclose the source and destination of all payments processed by the institution. Because of the nature of the XRP Ledger, anyone could potentially send XRP (and, under the right circumstances, issued currencies) to an institution's account on the XRP Ledger. Dealing with such unwanted payments adds significant overhead cost to these institutions' compliance departments, including potential fines or penalties.
```

Comment:
Perhaps strike 'overhead', as synonymous with cost:
> **Problem:** To comply with regulations like [BSA, KYC, AML, and CFT](https://ripple.com/build/gateway-guide/#gateway-compliance), financial institutions must provide documentation about the source of funds they receive. Such regulations seek to prevent the illicit transfer of funds by requiring institutions to disclose the source and destination of all payments processed by the institution. Because of the nature of the XRP Ledger, anyone could potentially send XRP (and, under the right circumstances, issued currencies) to an institution's account on the XRP Ledger. Dealing with such unwanted payments adds significant cost to these institutions' compliance departments, including potential fines or penalties.

Reply:
I agree that "overhead" and "cost" are synonymous, but they're not the same thing. I think specifying the _type_ of cost here makes the problem clearer. Open to arguments against that, though!

Reply:
Hmm, I think the truth of it is that my lack of understanding how expenses are categorized keeps me from agreeing that fees and penalties add to overhead cost. I'd encourage the specificity, as long as it's accurate. Rather, I think fees and penalties would be categorized differently, as they don't recur. I believe that overhead describes things like labor  and rent.